xwiki-rendering-syntaxes/xwiki-rendering-syntax-wikimodel/src/main/java/org/xwiki/rendering/internal/parser/wikimodel/WikiModelParserUtils.java

```diff
@@ -49,34 +75,8 @@ public void parseInline(StreamParser parser, String content, Listener listener,
         throws ParseException
     {
         if (prefix) {
-            WrappingListener inlineFilterListener = new InlineFilterListener()
-            {
-                private boolean foundWord;
-
-                private boolean foundSpace;
-
-                @Override
-                public void onWord(String word)
-                {
-                    if (this.foundWord) {
-                        super.onWord(word);
-                    } else {
-                        this.foundWord = true;
-                    }
-                }
-
-                @Override
-                public void onSpace()
-                {
-                    if (this.foundSpace) {
-                        super.onSpace();
-                    } else {
-                        this.foundSpace = true;
-                    }
-                }
-            };
+            WrappingListener inlineFilterListener = new PrefixIgnoredInlineFilterListener();
             inlineFilterListener.setWrappedListener(listener);
-
```

Comment:
Why remove that empty line ? I find it more readable to separate `inlineFilterListener` creation from the code using it.